strapi/strapi
# Hide Locale column and grouping option when i18n plugin is not installed

### What does it do?

It checks if the i18n plugin is installed, if not we need to hide the Locale column in the Release Details page and also the Grouping locale option.

### Why is it needed?

We want to replicate a similar behavior as in the CM. In the Release Details we can have entries from multiple collections so it is not possible to hide the column when a Collection is not using the i18n plugin (and also the entries can be grouped by Locale or actions).

### How to test it?

Check a Release Details page with the i18n plugin installed and then remove it with
```
yarn remove @strapi/plugin-i18n 
```
and re run the application, the locale option in the Grouping dropdown and the locale column need to be removed

### Related issue(s)/PR(s)

CS-549
